lib/presentation/pages/trip_panel_page/widgets/expense_tile_list.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+import 'package:trip_tally/presentation/pages/trip_panel_page/widgets/expense_tile.dart';
+import 'package:trip_tally/presentation/theme/app_paths.dart';
+
+List<Widget> getExpenseTiles() {
```

Comment:
if this is a mock, keep it in the file you use it, it will be easier to delete it later

Reply:
do i not?

Reply:
This is a separate file that has only this method right?